documentation/optee_design.md

```diff
@@ -451,6 +451,36 @@ OP-TEE to be able to initialize at all. The loader supplies in `r0/x0` the
 address of the first byte following what was not copied and jumps to the load
 address to start OP-TEE.
 
+Extra image files are generated for loaders who support loading separate images.
+Legacy OP-TEE image tee.bin is split into several v2 images; typically header
```

Comment:
We put it here just after original header to show difference; how about the section movement into another PR ?